README.md

```diff
@@ -19,17 +22,35 @@ You can chat and discuss the development of Thyme on our [Discord channel](https
 
 ## Running Thyme
 
-Currently Thyme can only be run on Windows.
 
-To run Thyme, you first need to have Command & Conquer Generals: Zero Hour installed. If you don't have the original game disc, Generals: Zero Hour can currently be bought most easily as a downloadable version that is part of the Command & Conquer The Ultimate Collection from [Origin](https://www.origin.com/en-us/store/command-and-conquer/command-and-conquer-the-ultimate-collection/ultimate-collection) or a third-party like [Amazon](https://www.amazon.com/Command-Conquer-Ultimate-Collection-PC/dp/B0095C0I4W).
+**As of now (Feb. 2023) Thyme only runs on Windows and must be compiled by yourself.**
```

Comment:
There actually is a pre-compiled thyme, which another contributor has attempted to highlight with change #815.

Reply:
An this does work as expected?
I mean if thats the case I'd ask myself why I did the whole compiling thing and nobody told me on DC.
Also OmniBlade stated that the binary is very limited in functionality, so would that really be helpful for the normal user?

Reply:
Well if it's working and wanted I can include this in the readme, just let me know